LiteCore/Query/QueryParser.cc

```diff
@@ -286,9 +286,9 @@ namespace litecore {
     }
 
 
-    void QueryParser::writeCreateIndex(const Array *expressions) {
+    void QueryParser::writeCreateIndex(const string &name, const Array *expressions) {
         reset();
-        _sql << "CREATE INDEX IF NOT EXISTS \"" << indexName(expressions) << "\" ON " << _tableName << " ";
+        _sql << "CREATE INDEX IF NOT EXISTS \"" << name << "\" ON " << _tableName << " ";
```

Comment:
In that case I think I should mimic the full text behavior and drop the first index in favor of the new one.  Is there a way to tell what the existing index is on without causing headaches?